xds/go13.go

```diff
@@ -0,0 +1,25 @@
+// +build go1.13
+
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package xds
+
+import (
+	_ "google.golang.org/grpc/credentials/tls/certprovider/meshca" // Register the MeshCA certificate provider plugin.
```

Comment:
Should we move this back after #4051?

Reply:
The xds interop should, but not the non-xds interop client.

https://github.com/grpc/grpc-go/blob/master/interop/client/client.go
vs
https://github.com/grpc/grpc-go/blob/master/interop/xds/client/client.go

I don't know what Go runtime we use for the xds tests, but I suspect it's more modern.  @ericgribkoff do you know the answer to this?

Reply:
It's what `apt get` gives us. Seems to be `go version go1.11.6 linux/amd64`

Reply:
Can this be changed?